[2024-12-05T22:54:20.187+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T22:54:20.200+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: brand_category_mapping.call_llm manual__2024-12-05T22:54:15.308681+00:00 [queued]>
[2024-12-05T22:54:20.207+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: brand_category_mapping.call_llm manual__2024-12-05T22:54:15.308681+00:00 [queued]>
[2024-12-05T22:54:20.207+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-05T22:54:20.219+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): call_llm> on 2024-12-05 22:54:15.308681+00:00
[2024-12-05T22:54:20.226+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8856) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T22:54:20.227+0000] {standard_task_runner.py:72} INFO - Started process 8857 to run task
[2024-12-05T22:54:20.227+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'brand_category_mapping', 'call_llm', 'manual__2024-12-05T22:54:15.308681+00:00', '--job-id', '1926', '--raw', '--subdir', 'DAGS_FOLDER/llm_mapping.py', '--cfg-path', '/tmp/tmpuwck0e8a']
[2024-12-05T22:54:20.229+0000] {standard_task_runner.py:105} INFO - Job 1926: Subtask call_llm
[2024-12-05T22:54:20.265+0000] {task_command.py:467} INFO - Running <TaskInstance: brand_category_mapping.call_llm manual__2024-12-05T22:54:15.308681+00:00 [running]> on host c7780884450d
[2024-12-05T22:54:20.330+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='brand_category_mapping' AIRFLOW_CTX_TASK_ID='call_llm' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T22:54:15.308681+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T22:54:15.308681+00:00'
[2024-12-05T22:54:20.331+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T22:54:20.349+0000] {llm_mapping.py:155} INFO - Извлечён return_value: {'batch': [], 'next_index': 5775}
[2024-12-05T22:54:20.350+0000] {llm_mapping.py:98} INFO - Пустой батч. Ничего не делать.
[2024-12-05T22:54:20.350+0000] {python.py:240} INFO - Done. Returned value was: 
[2024-12-05T22:54:20.369+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T22:54:20.370+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=brand_category_mapping, task_id=call_llm, run_id=manual__2024-12-05T22:54:15.308681+00:00, execution_date=20241205T225415, start_date=20241205T225420, end_date=20241205T225420
[2024-12-05T22:54:20.401+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T22:54:20.422+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T22:54:20.426+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
